packages/squiggle-lang/src/rescript/utility/E.res

```diff
@@ -269,6 +269,7 @@ module A = {
       ))
       |> Rationale.Result.return
     }
+  let rangeFloat = (start, stop) => start -> Belt.Array.rangeBy(stop, ~step=1) -> (arr => fmap(Belt.Int.toFloat, arr))
```

Comment:
nitpick, but I assume the last part could be shortened to: ``->(fmap(Belt.Int.toFloat))`` or similar. (You could also use |> if that is the difference. I made fmap2 recently, in a different PR)

Reply:
Nope -- pipe shifts positionality of arguments in the more counterintuitive way.

Reply:
I want to follow the guidelines in the docs, the ones where `|>` is deprecated, and not write any more of it. In fact we may want to go and do the annoying refactor of switching all the `|>` into `->` before alpha.